test/Microsoft.Extensions.SecretManager.Tests/ConfigurationExtensionTests.cs

```diff
@@ -18,7 +19,7 @@ public void AddUserSecrets_Does_Not_Fail_On_Non_Existing_File_Explicitly_Passed(
                                 .AddUserSecrets(userSecretsId: Guid.NewGuid().ToString());
         }
 
-        [Fact]
+        [ConditionalFact (Skip = "Obsolete API")]
```

Comment:
Actually I don't think there is much value keeping these tests around. Why not just remove them?